src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -466,9 +466,9 @@ protected override TDeclarationNode AddMembers<TDeclarationNode>(TDeclarationNod
             CodeGenerationOptions options,
             CancellationToken cancellationToken)
         {
-            if (destinationMember is MemberDeclarationSyntax memberDeclaration)
+            if (destinationMember is MemberDeclarationSyntax || destinationMember is LocalFunctionStatementSyntax)
```

Comment:
this doesn't seem necessary.  as before, if the destination is a local-function, we shoudl be called 'AddStatements` instead.  this feels lke this is masking an issue on the caller side.